<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmSensorOscillo</class>
 <widget class="QMainWindow" name="FrmSensorOscillo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_toolTrack">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>51</width>
      <height>670</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="toolButton_toolCrosshair">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>30</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Con trỏ chữ thập</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border-color: rgb(210, 210, 210);
border-style: outset;
border-width: 2px;
border-radius: 10px;
}
QToolButton:hover{
border-width: 0px;
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../chart.qrc">
       <normaloff>:/icon/chart/iconChartCrosshair_Off.jpg</normaloff>
       <activeoff>:/icon/chart/iconChartCrosshair_On.jpg</activeoff>:/icon/chart/iconChartCrosshair_Off.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_toolCallout">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>80</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Bắt điểm trên đường đồ thị</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border-color: rgb(210, 210, 210);
border-style: outset;
border-width: 2px;
border-radius: 10px;
}
QToolButton:hover{
border-width: 0px;
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../chart.qrc">
       <normaloff>:/icon/chart/iconChartCallout_Off.jpg</normaloff>
       <activeoff>:/icon/chart/iconChartCallout_On.jpg</activeoff>:/icon/chart/iconChartCallout_Off.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>0</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(25, 149, 173);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Con trỏ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_toolHideSeries">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>130</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Ẩn hiện đồ thị</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border-color: rgb(210, 210, 210);
border-style: outset;
border-width: 2px;
border-radius: 10px;
}
QToolButton:hover{
border-width: 0px;
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../chart.qrc">
       <normaloff>:/icon/chart/iconSeriesCheck_Off.jpg</normaloff>
       <activeoff>:/icon/chart/iconSeriesCheck_On.jpg</activeoff>:/icon/chart/iconSeriesCheck_Off.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsViewChart">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>1151</width>
      <height>670</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_Control">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>669</y>
      <width>1201</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_scope">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
background-color: rgb(250, 255, 255);
background-color: rgb(255, 255, 255);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 10px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="toolButton_plus">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>31</width>
        <height>70</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Xóa thiết bị</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton#toolButton_plus{
color: rgb(2, 87, 122);
background-color: rgb(25, 149, 173);
border: none;
border-style: outset;
border-width: 0px;
border-top-right-radius: 10px;
border-bottom-right-radius: 10px;
}
QToolButton#toolButton_plus:hover{
background-color: rgb(25, 149, 173);
background-color: rgb(25, 149, 190);
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../chart.qrc">
        <normaloff>:/icon/chart/iconChartPlus_On.jpg</normaloff>:/icon/chart/iconChartPlus_On.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_minus">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>31</width>
        <height>70</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Xóa thiết bị</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton#toolButton_minus{
color: rgb(2, 87, 122);
background-color: rgb(25, 149, 173);
border: none;
border-style: outset;
border-width: 0px;
border-top-left-radius: 10px;
border-bottom-left-radius: 10px;
}
QToolButton#toolButton_minus:hover{
background-color: rgb(25, 149, 173);
background-color: rgb(25, 149, 190);
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../chart.qrc">
        <normaloff>:/icon/chart/iconChartMinus_On.jpg</normaloff>:/icon/chart/iconChartMinus_On.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(3, 87, 123);</string>
      </property>
      <property name="text">
       <string>Phạm vi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_scope">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 170, 255);
</string>
      </property>
      <property name="text">
       <string>100ms/div</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_play">
     <property name="geometry">
      <rect>
       <x>1020</x>
       <y>50</y>
       <width>172</width>
       <height>73</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QToolButton" name="toolButton_toolPause">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>100</width>
        <height>71</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Dừng lại (Esc)</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 0px;
}
QToolButton:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../chart.qrc">
        <normaloff>:/icon/chart/iconChartPause_Off.jpg</normaloff>
        <activeoff>:/icon/chart/iconChartPause_On.jpg</activeoff>:/icon/chart/iconChartPause_Off.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="shortcut">
       <string>Esc</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_toolPlay">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>100</width>
        <height>71</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Bắt đầu (F1)</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 0px;
}
QToolButton:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../chart.qrc">
        <normaloff>:/icon/chart/iconChartPlay_Off.jpg</normaloff>
        <activeoff>:/icon/chart/iconChartPlay_On.jpg</activeoff>:/icon/chart/iconChartPlay_Off.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
      <property name="shortcut">
       <string>F1</string>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_toolClear">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>1</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Xóa đồ thị</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton{
color: rgb(2, 87, 122);
background-color: rgb(240, 240, 240);
border: none;
border-style: outset;
border-width: 0px;
border-radius: 0px;
}
QToolButton:hover{
background-color: rgb(25, 149, 173);
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../chart.qrc">
        <normaloff>:/icon/chart/iconChartClear_Off.jpg</normaloff>
        <activeoff>:/icon/chart/iconChartClear_On.jpg</activeoff>:/icon/chart/iconChartClear_Off.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>0</y>
        <width>3</width>
        <height>73</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_toolSetup">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#frame_toolSetup{
	background-color: rgb(25, 149, 173);
    /*background-color: rgb(3, 87, 123);;*/
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../chart.qrc"/>
 </resources>
 <connections/>
</ui>
